cartoframes/data/utils.py

```diff
@@ -0,0 +1,150 @@
+import time
```

Comment:
I really like this file, making the rest cleaner

Reply:
:heart: